mountain-climbing/src/main/java/com/example/mountainclimbing/controller/ImageController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.mountainclimbing.controller;
+
+import java.util.Optional;
+
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.mountainclimbing.model.Image;
+import com.example.mountainclimbing.service.ImageService;
+
```

Comment:
Add for each method return value ResponseEntity. According return value return ResponseEntity with current status. For example:
1. 201 - Created
2. 200 - Ok
3. 404 - Not found
4. etc